### PR TITLE
Fixup specs

### PR DESCRIPTION
**Pull request by sebastianv1**

### CODE REVIEW

**Comment by dnkoutso** [comment]
For https://github.com/CocoaPods/CocoaPods/pull/8339

File: install_remove_pod/before/SampleApp.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 /* Begin PBXBuildFile section */
 		0AEC166B36EA4BBAB35AEB94 /* libPods.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 061523DA9D2646ACA1EF295C /* libPods.a */; };
```
Comment (dnkoutso) [inline install_remove_pod/before/SampleApp.xcodeproj/project.pbxproj]: this is a big nit but this should not be there. I think we should have run `pod deintegrate` on this project to clean it up.

File: install_remove_pod/before/SampleApp.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -69,6 +85,7 @@
 				E575589E16C5943000DC1500 /* Cocoa.framework */,
 				E57558A016C5943000DC1500 /* Other Frameworks */,
 				061523DA9D2646ACA1EF295C /* libPods.a */,
```
Comment (dnkoutso) [inline install_remove_pod/before/SampleApp.xcodeproj/project.pbxproj]: same for this entry

File: install_remove_pod/before/SampleApp.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -168,19 +185,27 @@
...
 			files = (
 			);
+			inputFileListPaths = (
+			);
 			inputPaths = (
+				"${PODS_PODFILE_DIR_PATH}/Podfile.lock",
+				"${PODS_ROOT}/Manifest.lock",
+			);
+			name = "[CP] Check Pods Manifest.lock";
+			outputFileListPaths = (
```
Comment (dnkoutso) [inline install_remove_pod/before/SampleApp.xcodeproj/project.pbxproj]: hm I do not know how strict we are with this but this now depends on Xcode 10.x

File: install_remove_pod/before/SampleApp.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -40,18 +44,30 @@
 			files = (
 				E575589F16C5943000DC1500 /* Cocoa.framework in Frameworks */,
 				0AEC166B36EA4BBAB35AEB94 /* libPods.a in Frameworks */,
```
Comment (dnkoutso) [inline install_remove_pod/before/SampleApp.xcodeproj/project.pbxproj]: this one too

File: install_remove_pod/after/execution_output.txt [hunk 5]
```diff
@@ -7,35 +7,16 @@ Inspecting targets to integrate
   Using `ARCHS` setting to build architectures of target `Pods-SampleApp`: (`$(ARCHS_STANDARD_64_BIT)`)
 
 Finding Podfile changes
-  R JSONKit
```
Comment (segiddins) [inline install_remove_pod/after/execution_output.txt]: I dont think this is right, given this integration spec is supposed to be testing removing a pod

File: update_all/after/execution_output.txt [hunk 6]
```diff
@@ -13,26 +13,11 @@ Finding Podfile changes
...
-     You are in 'detached HEAD' state. You can look around, make experimental
-     changes and commit them, and you can discard any commits you make in this
-     state without impacting any branches by performing another checkout.
-
-     If you want to create a new branch to retain commits you create, you may
-     do so (now or later) by using -b with the checkout command again. Example:
-
-       git checkout -b new_branch_name
-
+-> Using Reachability (3.1.0)
```
Comment (segiddins) [inline update_all/after/execution_output.txt]: I dont think this is right either, we're no longer testing that we're updating anything

**Pull request merged**